backend/src/src-vulnerabilities/src/main/java/org/eclipse/sw360/vulnerabilities/db/VulnerabilityDatabaseHandler.java

```diff
@@ -35,22 +36,13 @@
 
     private static final Logger log = Logger.getLogger(VulnerabilityDatabaseHandler.class);
 
-    /**
-     * Connection to the couchDB database
-     */
-    private final DatabaseConnector db;
     private final VulnerabilityRepository vulRepo;
     private final VulnerabilityRelationRepository relationRepo;
 
-    public VulnerabilityDatabaseHandler(Supplier<HttpClient> httpClient, String dbName) throws MalformedURLException {
-        // Create the connector
-        db = new DatabaseConnector(httpClient, dbName);
-        vulRepo = new VulnerabilityRepository(db);
-        relationRepo = new VulnerabilityRelationRepository(db);
-    }
+    public VulnerabilityDatabaseHandler() throws MalformedURLException {
+        DatabaseConnector db = new DatabaseConnector(DatabaseSettings.getConfiguredHttpClient(),
+                DatabaseSettings.COUCH_DB_VM);
 
-    public VulnerabilityDatabaseHandler(DatabaseConnector db) throws MalformedURLException {
-        this.db = db;
         vulRepo = new VulnerabilityRepository(db);
         relationRepo = new VulnerabilityRelationRepository(db);
```

Comment:
Does that now mean, that the `VulnerabilityRealtions` are written to the `COUCH_DB_VM`? They should be written to the default database.

Reply:
Exactly. I documented the reasons in the bug ticket. That is a bit of a problem when having issue tickets and PRs :-)